piq/base.py

```diff
@@ -50,8 +50,14 @@ def compute_feats(
             # TODO(jamil 26.03.20): Add support for more than one feature map
             assert len(features) == 1, \
                 f"feature_encoder must return list with features from one layer. Got {len(features)}"
-            total_feats.append(features[0].view(N, -1))
 
+            features = features[0].view(N, -1)
+            features = features.cpu()
+            total_feats.append(features)
```

Comment:
Why not use one-liner `total_feats.append(features[0].view(N, -1).cpu())`?

Reply:
For better readability.